bluebottle/wallposts/admin.py

```diff
@@ -11,20 +14,36 @@
 from bluebottle.wallposts.models import SystemWallpost
 from .models import Wallpost, MediaWallpost, TextWallpost, MediaWallpostPhoto, Reaction
 
+from sorl.thumbnail.shortcuts import get_thumbnail
+
 
-class MediaWallpostPhotoInline(AdminImageMixin, admin.StackedInline):
+class MediaWallpostPhotoInline(admin.TabularInline):
     model = MediaWallpostPhoto
     extra = 0
-    raw_id_fields = ('author', 'editor')
+    raw_id_fields = ('author', 'editor') # for performance reasons?
+
+    readonly_fields = ('image_tag',)
+
+    fields = ('image_tag', 'photo')
+
+    def image_tag(self, obj):
+        data = {}
+        if obj.photo:
+            data['image_full_url'] = obj.photo.url
+            data['image_thumb_url'] = get_thumbnail(obj.photo, "120x120", crop="center").url
 
+        return render_to_string("admin/wallposts/mediawallpost_photoinline.html", data)
+
+    image_tag.short_description = 'Preview'
+    image_tag.allow_tags = True
 
 class MediaWallpostAdmin(PolymorphicChildModelAdmin):
     base_model = Wallpost
-    readonly_fields = ('ip_address', 'deleted')
+    readonly_fields = ('ip_address', 'deleted', 'view_online', 'gallery')
     raw_id_fields = ('author', 'editor')
-    list_display = ('created', 'view_online', 'get_text', 'video_url', 'photos', 'author')
+    list_display = ('created', 'view_online', 'get_text', 'thumbnail', 'author')
 
-    readonly_fields = ('view_online', )
+    extra_fields = ("gallery", )
```

Comment:
We try to use double quotes for translatable strings and single ones for things like this one. However you might find mix-ups in other parts of our code too. Not that important, but for consistency with the definitions above single quotes are better.